### PR TITLE
add configuration for report generation

### PR DESCRIPTION
**Pull request by ymhari**
ME-180268920

### CODE REVIEW

File: config/client_config/dc/system/config/templates/features/enroll_app/enroll_app.yml [hunk 1]
```diff
@@ -494,6 +494,9 @@ registry:
             item: false
           - key: :carriers_long_name
             item: 'Community Health Options'
+      - key: :carrier_abbev_list
```
Comment (markgoho) [inline config/client_config/dc/system/config/templates/features/enroll_app/enroll_app.yml:497]: is `abbev` supposed to be `abbrev`?

File: spec/lib/tasks/pay_now_click_tracking_spec.rb [hunk 2]
```diff
@@ -28,25 +28,51 @@
       )
     end
 
-    before do
-      payment_transaction1.reload
-      payment_transaction2.reload
+    context 'carrier for dc' do
```
Comment (markgoho) [inline spec/lib/tasks/pay_now_click_tracking_spec.rb:31]: there should be no mention of clients in spec files -- please refactor to generalize these specs

File: spec/lib/tasks/pay_now_click_tracking_spec.rb [hunk 3]
```diff
@@ -28,25 +28,51 @@
...
-      data = CSV.read "#{Rails.root}/public/pay_now_click_tracking.csv"
-      expect(data).to eq result
+      it "should generate the click tracking data" do
+        result = [["source", "enrollment_id", "datetime_of_click", "hbx_id"], ["plan_shopping","123456789","02/01/2021 00:00", "123456789"], ["enrollment_tile","123456789","02/01/2021 00:00", "123456789"]]
+        data = CSV.read "#{Rails.root}/public/pay_now_click_tracking.csv"
+        expect(data).to eq result
+      end
     end
 
+    context 'carrier for me' do
```
Comment (markgoho) [inline spec/lib/tasks/pay_now_click_tracking_spec.rb:54]: same as above, specs should be client-agnostic

**Review by markgoho — CHANGES_REQUESTED** [review]

**Comment by markgoho** [comment]
@ymhari please add a ticket reference to the comment box at the top

**Review by markgoho — APPROVED** [review]
changes look good, thanks @ymhari 👍🏻

**Review by RyanEddyIC — APPROVED** [review]

**Pull request merged**